src/wasm-s-parser.h

```diff
@@ -60,6 +60,7 @@ class Element {
 
   size_t line, col;
   SourceLocation* loc;
+  SourceLocation* endLoc;
```

Comment:
end of what?

Reply:
`loc` contains original source location for beginning of s-expr parents/block, `endLoc` contains original source location for ending of this block.

Reply:
Oh ok, then how about `startLoc, endLoc`?

Reply:
Plus a comment could help here I think.